src/content/changes/highlights/2023-07-27.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2023-07-27T14:00:00
+title: Highlights for the week ending July 27 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Nginx Ingress Controller 
+We have spent the last months on reworking our [nginx-ingress-controller-app](https://github.com/giantswarm/ingress-nginx-app) by aligning it to the [upstream ingress-nginx]u(https://github.com/kubernetes/ingress-nginx) project. This was necessary to be future proof, feature compliant and to offer the best ingress experience possible. Therefor we want to announce the first public beta release of our new `ingress-nginx chart`. At the moment it is not recommended to install this release in a production environment, it is for testing purposes only. The stable release v3.0.0 will be cut as soon as all possible bugs have been fixed and improvements have been implemented. For our  [beta release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-beta1) we added all changes mentioned in our [alpha1 release](https://github.com/giantswarm/ingress-nginx-app/releases/tag/v3.0.0-alpha1).
+
+To have a flawless experience we wrote a [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0-beta1/migration.md). This guide contains a step-by-step guide which replaces your old app deployment to upgrade the chart without any major outages. 
+
+All feedback and questions regarding the beta relase, its changes and our migration guide are very welcome.
+
+<span style="color:red">Warning: Do not install the beta release in production environments.</span>.
+
+## Apps
+
+- [flux](https://github.com/giantswarm/flux-app) version [v0.24.0](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0240---2023-07-04) This change updates the default `securityContext` values to comply with our Pod Security Standard policies.
+- [crossplane](https://github.com/giantswarm/crossplane) version [v2.3.0](https://github.com/giantswarm/crossplane) We updated Crossplane to [v1.12.2](https://github.com/crossplane/crossplane/releases/tag/v1.12.2) Please check the [Crossplane Changelog](https://github.com/crossplane/crossplane/releases/tag/v1.12.2) for all changes
+- [external-secrets](https://github.com/giantswarm/external-secrets) version [v0.6.4](https://github.com/giantswarm/external-secrets) In prepration to have cilium CNI running we have split the original and cilium network policies instead of excluding each other by conditiion.
```

Comment:
```suggestion
- [flux](https://github.com/giantswarm/flux-app) version [v0.24.0](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0240---2023-07-04). We have improved the security by updating the default `securityContext` values to comply with our Pod Security Standard policies.
- [crossplane](https://github.com/giantswarm/crossplane) version [v2.3.0](https://github.com/giantswarm/crossplane) We updated Crossplane to [v1.12.2](https://github.com/crossplane/crossplane/releases/tag/v1.12.2) Please check the [Crossplane Changelog](https://github.com/crossplane/crossplane/releases/tag/v1.12.2) for all changes announced. 
- [external-secrets](https://github.com/giantswarm/external-secrets) version [v0.6.4](https://github.com/giantswarm/external-secrets). In preparation to have cilium CNI running we have split the original and cilium network policies instead of excluding each other by condition.
```

> In preparation to have cilium CNI running we have split the original and cilium network policies instead of excluding each other by condition.

I have tried to get what we have done here but I could not understand (I have looked to the PR too). Why do we install two times the CNP to target same labels when there are CRDs enabled? @uvegla

Reply:
Because there were some use cases when the cilium CRDs were on the cluster but not the controllers. Since previously they were exclusive it did install the cilium policy but nothing was present to control them so the app was not working.

NOTE: the `external-secrets` version link is not linking to the version / changelog sections.